xml/dm_registers.xml

```diff
@@ -78,19 +86,27 @@
 
         <!-- Fields that apply to the entire DM. -->
         <field name="authenticated" bits="7" access="R" reset="Preset">
-            0: Authentication is required before using the DM.
+            <value v="0" name="required">
```

Comment:
This seems more like authenticated/unauthenticated or authenticated=true/false than required/passed.

Reply:
Changed to false/true.